mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionProvider.scala

```diff
@@ -275,47 +240,38 @@ class CompletionProvider(
       val suffix = v.snippetSuffix
       path match
         case (_: Ident) :: (_: Import) :: _ =>
-          mkWorkspaceItem(sym.fullNameBackticked)
+          mkItem(sym.fullNameBackticked)
         case _ =>
           autoImports.editsForSymbol(sym) match
             case Some(edits) =>
               edits match
                 case AutoImportEdits(Some(nameEdit), other) =>
-                  mkItem0(
```

Comment:
Do you think we can move `editsForSymbol` to where we create the completion?

Reply:
That's the plant, but I didn't want to do the refactor all at once.